keep-ui/app/navbar-inner.tsx

```diff
@@ -210,30 +221,32 @@ export default function NavbarInner({ user }: { user?: User }) {
                       leaveTo="transform opacity-0 scale-95"
                     >
                       <Menu.Items className="absolute right-0 z-10 mt-2 w-48 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
-                        <Menu.Item>
-                          {({ active }) => (
-                            <>
-                              <a
-                                className={classNames(
-                                  "flex w-full px-4 py-2 text-sm text-gray-700 hover:bg-gray-100"
-                                )}
-                                href="/settings"
-                              >
-                                Settings
-                              </a>
-                              {authType != AuthenticationType.NO_AUTH ? (
-                                <button
+                          <Menu.Item>
+                            {({ active }) => (
+                              <>
+                              {!isNocRole &&
```

Comment:
I think it's better not to create the `MenuItem` at all
```suggestion
                          {!isNocRole && <Menu.Item>
                            {({ active }) => (
                              <>
```